### ISSUE TITLE
Error in python test for macOS

### ISSUE THREAD

**Issue by Davtax** [Davtax/arXiv-sorter#15]
Sometimes, when performing the Python test with the GitHub actions for macOS, it fails in the `check_version` part. This error is caused by the line `new_version = response.json()['tag_name']`, showing the next message.
```
KeyError: 'tag_name'
Error: Process completed with exit code 1.
```
I have not been able to find an explanation for this behavior. When rerunning the action, it is usually solved.

**Comment by Davtax** [comment]
A more detailed error is:
`{'message': "API rate limit exceeded for 13.105.117.39. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)", 'documentation_url': 'https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting'}`

**Comment by Davtax** [comment]
Solved in #17.
